requirements.txt

```diff
@@ -41,6 +41,7 @@ terminaltables==3.1.0
 sanic==19.9.0
 sanic-cors==0.9.9.post1
 sanic-jwt==1.3.2
+sanic-plugins-framework==0.8.2
```

Comment:
Do you also want to add a link to the PR?

Reply:
to which pr? added one in the description of this pr

Reply:
@federicotdn asked me to add `# https://github.com/RasaHQ/rasa/pull/5064` before that line.

Reply:
done 👍